README.md

```diff
@@ -10,6 +19,11 @@ The package is published on PyPI. Install with :
 
 ## Usage 
 
+**User story**
+As a `fr-format` user, specifically a developer, you may need to validate a value (which must be of type number(integer/float) or string) according to a given French format. This feature enables you to easily verify whether the data provided by your project users is valid or not.\
+Additionally, this project allows you to retrieve the entire set of valid values, depending on the format and the version(if it exists) used. This can be particularly useful in your frontend, for example, to display the complete set.
```

Comment:
```suggestion
Additionally, this project allows you to retrieve the entire set of valid values when applicable. For example, this can be useful in a frontend to display the complete set (e.g. in a dropdown menu).
```